templates/ticket/edit-notes.html

```diff
@@ -1,5 +1,5 @@
-<form id="edit-notes" action="" method="post">
+<form id="edit-notes" action="./notes" method="post">
```

Comment:
Please use `reverse` instead of relative links. We do not rely on URL hierarchy and always reverse the URL.